tests/common/devices/sonic.py

```diff
@@ -45,7 +45,7 @@ def __init__(self, ansible_adhoc, hostname,
                  ssh_user=None, ssh_passwd=None):
         AnsibleHostBase.__init__(self, ansible_adhoc, hostname)
 
-        self.DEFAULT_ASIC_SERVICES = ["bgp", "database", "lldp", "swss", "syncd", "teamd"]
+        self.DEFAULT_ASIC_SERVICES = ["bgp", "database", "lldp", "swss", "syncd", "teamd", "gbsyncd"]
```

Comment:
`gbsyncd` is not applicable for all the platforms. so I think we can add it to `DEFAULT_ASIC_SERVICES` on when the feature is enabled.

Reply:
thanks @arlakshm for pointer! fixed